specs/rollup-node.md

```diff
@@ -268,6 +270,25 @@ previous section to each successive L1 block until we have caught up with the L1
 
 > **TODO** specify genesis block
 
+## L2 Output RPC method
+
+The Rollup node has its own RPC method, `optimism_output` which returns the
+a 32 byte hash corresponding to the [SSZ] encoded [L2Output](./proposals.md#l2-output-commitment-construction).
+
+[SSZ]: https://github.com/ethereum/consensus-specs/blob/dev/ssz/simple-serialize.md
+
+### Output Method API
+
+The input and return types here are as defined by the [engine API specs][engine-structures]).
+
+[engine-structures]: https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md#structures
+
+- method: `optimism_outputAtBlock`
+- params:
+  1. `QUANTITY` - L1 integer block number, or the strings `"safe"`, `"latest"`, or `"pending"`
+- returns:
+  2.`DATA` - The 32 byte output root
+
```

Comment:
Design questions:
- is there an argument that this should go on the execution engine? I prefer it on the rollup node because it can handle evaluating the fork choice.
- This is the first rollup node specific RPC method to be spec'd afaik. Is `optimism_` the right method prefix here?
- Should we allow querying this by either an L2 or L1 block height? If so, should we do that by:
  - having two optional params (`l1block` and `l2block`) or
  - two methods (`optimism_outputAtL1Block`, `optimism_outputAtL2Block`?

Reply:
I talked to @karlfloersch. The conclusion was: 
1. No.
2. Sure.
3. It should be queried by the l2Block number. No need to have multiple methods.